[2022-04-11 17:56:13,584] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: schedule_data.rm_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-04-11 17:56:13,594] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: schedule_data.rm_task scheduled__2019-01-01T00:00:00+00:00 [queued]>
[2022-04-11 17:56:13,595] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-11 17:56:13,595] {taskinstance.py:1239} INFO - Starting attempt 14 of 14
[2022-04-11 17:56:13,595] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-11 17:56:13,608] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): rm_task> on 2019-01-01 00:00:00+00:00
[2022-04-11 17:56:13,617] {standard_task_runner.py:52} INFO - Started process 346202 to run task
[2022-04-11 17:56:13,624] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'schedule_data', 'rm_task', 'scheduled__2019-01-01T00:00:00+00:00', '--job-id', '1702', '--raw', '--subdir', 'DAGS_FOLDER/nfl_to_gcs.py', '--cfg-path', '/tmp/tmpcpo3p79v', '--error-file', '/tmp/tmpvqei276y']
[2022-04-11 17:56:13,625] {standard_task_runner.py:77} INFO - Job 1702: Subtask rm_task
[2022-04-11 17:56:13,694] {logging_mixin.py:109} INFO - Running <TaskInstance: schedule_data.rm_task scheduled__2019-01-01T00:00:00+00:00 [running]> on host 4ed3eda454fd
[2022-04-11 17:56:13,770] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=schedule_data
AIRFLOW_CTX_TASK_ID=rm_task
AIRFLOW_CTX_EXECUTION_DATE=2019-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-01-01T00:00:00+00:00
[2022-04-11 17:56:13,772] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-11 17:56:13,772] {subprocess.py:74} INFO - Running command: ['bash', '-c', ' echo None']
[2022-04-11 17:56:13,787] {subprocess.py:85} INFO - Output:
[2022-04-11 17:56:13,790] {subprocess.py:89} INFO - None
[2022-04-11 17:56:13,791] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-11 17:56:13,819] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=schedule_data, task_id=rm_task, execution_date=20190101T000000, start_date=20220411T175613, end_date=20220411T175613
[2022-04-11 17:56:13,876] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-11 17:56:13,916] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
